test/e2e/file.bpmn

```diff
@@ -0,0 +1,249 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
:question:  this file seems not used, could we remove it?
otherwise same remark about the content as in the inline xml bpmn available directly in the test code

Reply:
sure